src/WaCore.Web/Middleware/SecureHeaders/SecureHeadersMiddlewareExtensions.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Microsoft.AspNetCore.Builder;
+using WaCore.Web.Middleware.SecureHeaders.Models;
+
+namespace WaCore.Web.Middleware.SecureHeaders
```

Comment:
It appears to be a common practice to put IApplicationBuilder extension methods in namespace Microsoft.AspNetCore.Builder so people can use them without adding a `using`